pandas/tests/groupby/test_function.py

```diff
@@ -400,6 +400,28 @@ def test_groupby_non_arithmetic_agg_int_like_precision(i):
         assert res.iloc[0].b == data["expected"]
 
 
+@pytest.mark.parametrize("func, expected", [
+    ("idxmin", {'c_int': [0, 2], 'c_float': [1, 3], 'c_date': [1, 2]}),
+    ("idxmax", {'c_int': [1, 3], 'c_float': [0, 2], 'c_date': [0, 3]})
+])
+def test_idxmin_idxmax_returns_int_types(func, expected):
+    # GH 25444
+    df = pd.DataFrame({'name': ['A', 'A', 'B', 'B'],
+                       'c_int': [1, 2, 3, 4],
+                       'c_float': [4.02, 3.03, 2.04, 1.05],
+                       'c_date': ['2019', '2018', '2016', '2017']})
+    df['c_date'] = pd.to_datetime(df['c_date'])
+
+    result = getattr(df.groupby('name'), func)()
+
+    for col in result:
```

Comment:
`assert_frame_equal` will check dtypes for you so no need for this as it is duplicative

Reply:
removed https://github.com/pandas-dev/pandas/pull/25531/commits/29afd36e88bc2e19ab04add6219f526fce69cfea